docs/rules/idiomatic/prefer-strings-count.md

```diff
@@ -0,0 +1,50 @@
+# prefer-strings-count
+
+**Summary**: Prefer to use `strings.count`
```

Comment:
question: Is there a difference between our 'use' and 'prefer' rules?

```
docs/rules/custom/prefer-value-in-head.md
3:**Summary**: Prefer value in rule head

docs/rules/imports/prefer-package-imports.md
3:**Summary**: Prefer importing packages over rules

docs/rules/idiomatic/prefer-set-or-object-rule.md
3:**Summary**: Prefer set or object rule over comprehension

docs/rules/idiomatic/boolean-assignment.md
3:**Summary**: Prefer `if` over boolean assignment

docs/rules/idiomatic/equals-pattern-matching.md
3:**Summary**: Prefer pattern matching in function arguments

docs/rules/style/prefer-some-in-iteration.md
3:**Summary**: Prefer `some .. in` for iteration

docs/rules/style/default-over-not.md
3:**Summary**: Prefer default assignment over negated condition

docs/rules/style/prefer-snake-case.md
3:**Summary**: Prefer snake_case for names

docs/rules/style/default-over-else.md
3:**Summary**: Prefer default assignment over fallback `else`

docs/rules/style/use-assignment-operator.md
3:**Summary**: Prefer `:=` over `=` for assignment
```

```
3:**Summary**: Use raw strings for regex patterns

docs/rules/idiomatic/use-if.md
3:**Summary**: Use the `if` keyword

docs/rules/idiomatic/use-some-for-output-vars.md
3:**Summary**: Use `some` to declare output variables

docs/rules/idiomatic/use-in-operator.md
3:**Summary**: Use `in` to check for membership

docs/rules/idiomatic/use-contains.md
3:**Summary**: Use the `contains` keyword

docs/rules/imports/use-rego-v1.md
3:**Summary**: Use `import rego.v1`

docs/rules/bugs/not-equals-in-loop.md
3:**Summary**: Use of != in loop
```

Still working out how I feel, but I think 'Use' is more opinionated and clear than 'Prefer'.

I don't really want to update existing rules, but I think this new rule might be a 'Use' rule since there is no preference about it, strings.Count is better right?

Reply:
👍 sounds good to me!